application/api/example/data.js

```diff
@@ -0,0 +1,9 @@
+({
+  access: 'public',
+
+  method: async () => {
+    return 'Hello world';
+  },
+
+  returns: 'string',
+});
```

Comment:
it could be one more variant
```
({
  access: 'public',

  method: (args) => {
    return new Promise((rv) => setTimeout(rv, 1000, args));
  },
});
```

Reply:
Nice example. But I think we should not change the contract